### PR TITLE
Sample Code for Chase Pay Transaction

### PR DESCRIPTION
**Pull request by sharathkatteri**

### CODE REVIEW

File: PaymentTransactions/CreateChasePayTransaction.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            };
+
+            var creditCard = new creditCardType
+            {
+                cardNumber = "4111111111111111",
+                expirationDate = "0725",
+                cardCode = "999",
+                // Set the token specific info
+                isPaymentToken = true,
+                cryptogram = "EjRWeJASNFZ4kBI0VniQEjRWeJA=",           // Set this to the value of the cryptogram received from the token provide
```
Comment (gnongsie) [inline PaymentTransactions/CreateChasePayTransaction.cs]: Place the comment in the line above.

File: PaymentTransactions/CreateChasePayTransaction.cs [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                tokenRequestorId = "12345678901",
+                tokenRequestorEci = "07"
+            };
+
+            //standard api call to retrieve response
+            var paymentType = new paymentType { Item = creditCard };
+
+            var transactionRequest = new transactionRequestType
+            {
+                transactionType = transactionTypeEnum.authCaptureTransaction.ToString(),    // charge the card
```
Comment (gnongsie) [inline PaymentTransactions/CreateChasePayTransaction.cs]: Remove comment

File: PaymentTransactions/CreateChasePayTransaction.cs [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                    else
+                    {
+                        Console.WriteLine("Error Code: " + response.messages.message[0].code);
+                        Console.WriteLine("Error message: " + response.messages.message[0].text);
+                    }
+                }
+            }
+            else
+            {
+                Console.WriteLine("Null Response.");
```
Comment (gnongsie) [inline PaymentTransactions/CreateChasePayTransaction.cs]: This line is misleading. There is a response if the API XSD validation fails.
Please refer to [CreateCustomerProfile sample code in Java](https://github.com/AuthorizeNet/sample-code-java/blob/master/src/main/java/net/authorize/sample/CustomerProfiles/CreateCustomerProfile.java), lines 73-78.

**Review by gnongsie — CHANGES_REQUESTED** [review]

File: PaymentTransactions/CreateChasePayTransaction.cs (hunk 1 above)
Comment (sharathkatteri) [reply PaymentTransactions/CreateChasePayTransaction.cs]: Made Changes

File: PaymentTransactions/CreateChasePayTransaction.cs (hunk 2 above)
Comment (sharathkatteri) [reply PaymentTransactions/CreateChasePayTransaction.cs]: Removed the comment

File: PaymentTransactions/CreateChasePayTransaction.cs (hunk 3 above)
Comment (sharathkatteri) [reply PaymentTransactions/CreateChasePayTransaction.cs]: Made necessary changes

**Review by gnongsie — APPROVED** [review]

**Comment by gnongsie** [comment]
Ready to merge

**Pull request merged**